internal/core/subsume/vertex.go

```diff
@@ -170,11 +173,11 @@ func (s *subsumer) vertices(x, y *adt.Vertex) bool {
 			continue
 		}
 
-		s.missing = f
 		s.gt = a
-		s.lt = y
+		s.lt = b
+
+		s.errf("%v%s does not subsume %v%s", a, a.ArcType.Suffix(), b, b.ArcType.Suffix())
```

Comment:
This has been modified to improve the error for situations like: `{foo:_1}_⊑_{foo?:_1}`. Currently the error is:

```
1 does not subsume 1
```

But there will now be a new error of

```
1 does not subsume 1?
```

This may be considered an abuse of syntax, but it's a starting point for discussion.